server/src/server.ts

```diff
@@ -130,8 +130,12 @@ export default function startServer(): void {
      */
     app.get('/api/v2/events', userIsAuthorized, (req, res) => getEvents(req, res));
     app.get('/api/v2/events/:id', userIsAuthorized, (req, res) => getEvent(req, res));
-    app.get('/api/v2/events/:event_id/eventregistrations', userIsAuthorized, (req, res) => getEventRegistration(req, res));
-    app.get('/api/v2/events/eventregistrations', userIsAuthorized, (req, res) => getEventRegistrations(req, res));
+    // Get a single event registration by eventId and userId
+    app.get('/api/v2/events/:event_id/eventregistration', userIsAuthorized, (req, res) => getEventRegistration(req, res));
```

Comment:
![grafik](https://user-images.githubusercontent.com/56634686/107692603-4f450f00-6cad-11eb-82c0-1f117b15ed10.png)
![grafik](https://user-images.githubusercontent.com/56634686/107692638-5b30d100-6cad-11eb-8175-22e0e1735745.png)
als nicht admin bekomme ich meine registartion nciht obwohl in DB vorhanden

Reply:
ja dummer fehler von mir. Danke! nice find.

Reply:
ja dummer fehler von mir. Danke! nice find.